internal/db/storage/blob/s3/session/session.go

```diff
@@ -68,5 +69,7 @@ func (s *sess) Session() (*session.Session, error) {
 		cfg = cfg.WithCredentials(creds)
 	}
 
+	cfg = cfg.WithDisableSSL(!s.enableSSL)
```

Comment:
oh... I didn't know about this, hmm how about this
```suggestion
	if !s.enableSSL {
		cfg = cfg.WithDisableSSL(true)
	}
```

Reply:
There are several cases using "WithDisableXXX" functions in AWS SDK. :fearful:
so do you think it is better to write like it for every cases?

Reply:
I think aws SDK's all feature is enabled by default, that's why they designed like that, but our concept is disabled by default because Go's boolean value is false by default. 
I think it's better to write like it for every case but I understand it's boring.

Reply:
okay. i'll do like that.

AWS SDK also has several "WithEnableXXX" functions. :rofl:
maybe they are thinking that all options should be "false" by default.

Reply:
okay, our final decision is we should use EnableXXX and don't use implicit default enable configuration.